wsd/DocumentBroker.hpp

```diff
@@ -1294,6 +1295,10 @@ class DocumentBroker : public std::enable_shared_from_this<DocumentBroker>
         bool isExport() const { return _isExport; }
         bool isRename() const { return _isRename; }
 
+        /// When the callback fires, we set this request as completed, so we destroy it.
+        bool completed() const { return _completed; }
+        void complete() { _completed = true; }
```

Comment:
isComplete() and setComplete() =) looks a tad confusing when reading the callers otherwise =)